### PR TITLE
Update OpenTelemetry/Instrumentation to 1.6.0/1.6.0

### PR DESCRIPTION
**Pull request by tnevolin**

### CODE REVIEW

File: CHANGELOG.md
Comment (carlosalberto) [inline CHANGELOG.md:4]: This is ok, although usually goes into the "Pending Release" section, in case we don't merge soon.

File: agent/pom.xml
Comment (carlosalberto) [inline agent/pom.xml:73]: One thing I've wanted to do is to simply re-use here `<opentelemetry.instrumentation.version>` but I wonder if there was a reason why Sergey (previous maintainer) did this. Maybe let's dig into that in a follow up.

**Review by carlosalberto — APPROVED** [review]

File: CHANGELOG.md
Comment (tnevolin) [reply CHANGELOG.md:4]: So you mean this should change only after merge?

**Pull request merged**
